src/library/Uno.Material/Styles/Application/AnimationConstants.xaml

```diff
@@ -7,4 +7,9 @@
 	<CubicEase x:Key="Material_EaseOutFunction"
 			   EasingMode="EaseOut" />
 
+	<Duration x:Key="Material_AnimationDuration">0:0:0.25</Duration>
+	<Duration x:Key="Material_AnimationDuration_Short">0:0:0.15</Duration>
+	<Duration x:Key="Material_AnimationDuration_Long">0:0:0.4</Duration>
+
+	<x:String x:Key="Material_AnimationDelay">0:0:0.15</x:String>
```

Comment:
Also, why is this is a `string`? Shouldn't it be `TimeSpan`?

Reply:
Never manage to get TimeSpan to work even on UWP so Chunk suggested using string instead and it does the trick.